src/force-release.ts

```diff
@@ -0,0 +1,160 @@
+        },
+        {
+          name: "Set git identity",
+          run: 'git config user.name "github-actions"\ngit config user.email "github-actions@github.com"',
+        },
+        {
+          name: "Setup Node.js",
+          uses: "actions/setup-node",
+          with: { "node-version": "16.14.0" },
+        },
+        {
+          name: "Install dependencies",
+          run: "yarn install --check-files --frozen-lockfile",
+        },
+        { name: "release", run: "npx projen release" },
+        {
+          name: "Backup artifact permissions",
+          run: "cd dist && getfacl -R . > permissions-backup.acl",
+          continueOnError: true,
+        },
+        {
+          name: "Upload artifact",
+          uses: "actions/upload-artifact",
+          with: { name: "build-artifact", path: "dist" },
+        },
+      ],
+    });
+
+    workflow.addJob("force_release_golang", {
+      name: "Publish to Github Go Repository",
+      needs: ["force-release"],
+      runsOn: workflowRunsOn,
+      permissions: {
+        contents: JobPermission.READ,
+      },
+      env: {
+        CI: "true",
+      },
+      steps: [
+        {
+          name: "Setup Node.js",
+          uses: "actions/setup-node",
+          with: { "node-version": "16.14.0" },
+        },
+        {
+          name: "Setup Go",
+          uses: "actions/setup-go",
+          with: { "go-version": "^1.16.0" },
+        },
+        {
+          name: "Download build artifacts",
+          uses: "actions/download-artifact",
+          with: { name: "build-artifact", path: "dist" },
+        },
+        {
+          name: "Restore build artifact permissions",
+          run: "cd dist && setfacl --restore=permissions-backup.acl",
+          continueOnError: true,
+        },
+        {
+          name: "Prepare Repository",
+          run: "mv dist .repo",
+        },
+        {
+          name: "Install Dependencies",
+          run: "cd .repo && yarn install --check-files --frozen-lockfile",
+        },
+        {
+          name: "Create Artifact",
+          run: "cd .repo && npx projen package:go",
+        },
+        {
+          name: "Setup Copywrite tool",
+          uses: "hashicorp/setup-copywrite",
+        },
+        {
+          name: "Copy copywrite hcl file",
+          run: "cp .repo/.copywrite.hcl .repo/dist/go/.copywrite.hcl",
+        },
+        {
+          name: "Add headers using Copywrite tool",
+          run: "cd .repo/dist/go && copywrite headers",
+        },
+        {
+          name: "Remove copywrite hcl file",
+          run: "rm -f .repo/dist/go/.copywrite.hcl",
+        },
+        {
+          name: "Collect go Artifact",
+          run: "mv .repo/dist dist",
+        },
+        {
+          name: "Release",
+          env: {
+            GIT_USER_NAME: "CDK for Terraform Team",
+            GIT_USER_EMAIL: "github-team-tf-cdk@hashicorp.com",
+            GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}",
```

Comment:
Does this token need more permissions?

```
      permissions: {
        contents: JobPermission.READ,
      },
```

Or maybe you want the Projen token here?

Reply:
Good catch. Updated.